README.md

```diff
@@ -60,47 +52,17 @@ Add an example of the use of the module in the following code block.
 Use real values instead of "var.<var_name>" or other placeholder values
 unless real values don't help users know what to change.
 -->
-
-#### Before You Begin
-If you are using this module to create an ATracker route and IBM Cloud Object Storage instance and bucket, and using a key from a key management service in a separate account, you will need an IAM authorization policy in the account where the key management service resides which grants the IBM Cloud Object Storage service in this account Reader access to the key management service.
-
 ```hcl
-locals {
-  at_endpoint = "https://api.us-south.logging.cloud.ibm.com"
-}
-
-provider "logdna" {
-  alias      = "at"
-  servicekey = ""
-  url        = local.at_endpoint
-}
-
-provider "logdna" {
-  alias      = "ld"
-  servicekey = ""
-  url        = local.at_endpoint
-}
-
-module "enterprise_account" {
+module "account_configuration" {
     source  = "terraform-ibm-modules/account-infrastructure-base/ibm"
     version = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
-    providers = { # providers block necessary for logdna provider aliases
-        logdna.at = logdna.at
-        logdna.ld = logdna.ld
-    }
-    region                            = "us-south"
     resource_group_name               = "account-base-resource-group"
-    provision_atracker_cos            = true # setting this enables provisioning of the ATracker + COS resources
-    cos_skip_iam_authorization_policy = false # setting this enables provisioning an authorization policy between the COS instances and the KMS instance given via the CRN
-    kms_key_crn                       = "crn:v1:bluemix:public:(kms|hs-crypto):(region):a/(Account ID):(KMS instance GUID)::"
-    cos_instance_name                 = "account-base-cos-instance"
-    cos_bucket_name                   = "atracker-cos-bucket"
-    cos_target_name                   = "atracker-cos-target"
     trusted_profile_name              = "account-base-trusted-profile"
-    activity_tracker_route_name       = "atracker-cos-route"
 }
 ```
 
+#### Before You Begin
```

Comment:
Can you remove this? This is not in our normal template and looks wrong:
![image](https://github.com/user-attachments/assets/01d0d8c5-89a9-46b1-b833-51f205a28e25)